roles/ansible_service_broker/tasks/install.yml

```diff
@@ -22,23 +22,14 @@
     ansible_service_broker_registry_user: "{{ ansible_service_broker_registry_user | default(__ansible_service_broker_registry_user) }}"
     ansible_service_broker_registry_password: "{{ ansible_service_broker_registry_password | default(__ansible_service_broker_registry_password) }}"
     ansible_service_broker_registry_organization: "{{ ansible_service_broker_registry_organization | default(__ansible_service_broker_registry_organization) }}"
-
-    ansible_service_broker_certs_dir: "{{ openshift.common.config_base }}/service-catalog"
+    ansible_service_broker_registry_tag: "{{ ansible_service_broker_registry_tag | default(__ansible_service_broker_registry_tag) }}"
+    ansible_service_broker_registry_whitelist: "{{ ansible_service_broker_registry_whitelist | default(__ansible_service_broker_registry_whitelist) }}"
 
 - name: set ansible-service-broker image facts using set prefix and tag
   set_fact:
     ansible_service_broker_image: "{{ ansible_service_broker_image_prefix }}ansible-service-broker:{{ ansible_service_broker_image_tag }}"
     ansible_service_broker_etcd_image: "{{ ansible_service_broker_etcd_image_prefix }}etcd:{{ ansible_service_broker_etcd_image_tag }}"
 
-- set_fact:
-    openshift_master_config_dir: "{{ openshift.common.config_base }}/master"
-  when: openshift_master_config_dir is undefined
-
-- slurp:
-    src: "{{ openshift_master_config_dir }}/service-signer.crt"
```

Comment:
So I see this was changed from master ca to service-signer ca, should I add this change back in and remove the below logic where I pull the service cert out of the asb_client secret?

Reply:
cc @staebler

Reply:
I don't have much of an opinion on this. It seems that elsewhere (at least within the service-catalog code) is using the files directly rather than grabbing from the secret. However, it seems less fragile to me, from a maintainability standpoint, to pull from the secret.